src/apps/Agreement/ConfigurationChecklist.js

```diff
@@ -0,0 +1,209 @@
+import React, { useEffect, useState } from 'react'
+import {
+  Box,
+  useLayout,
+  GU,
+  useTheme,
+  ProgressBar,
+  textStyle,
+  Button,
+  springs,
+} from '@aragon/ui'
+import PropTypes from 'prop-types'
+import checklistCompleteGraphic from './assets/checklist-complete.png'
+import { Transition, animated } from 'react-spring'
```

Comment:
Thought this was a good example with a couple of cases.

We would normally import in this order:

1. React
2. React-related (prop types, redux, etc)
3. External dependencies
4. External dependencies we control
5. Internal files

So it would look like this:

```suggestion
import React, { useEffect, useState } from 'react'
import PropTypes from 'prop-types'
import { Transition, animated } from 'react-spring'
import {
  Box,
  Button,
  ProgressBar,
  springs,
  textStyle,
  useLayout,
  useTheme,
  GU,
} from '@aragon/ui'
import checklistCompleteGraphic from './assets/checklist-complete.png'
```